dev/Slider/Slider_themeresources.xaml

```diff
@@ -210,26 +292,61 @@
-                                <VisualState x:Name="Normal" />
-
-                                <VisualState x:Name="Pressed">
-
+                                <VisualState x:Name="Normal">
+                                    <Storyboard>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="HorizontalThumb" Storyboard.TargetProperty="Background">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderThumbBackground}" />
+                                        </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="VerticalThumb" Storyboard.TargetProperty="Background">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderThumbBackground}" />
+                                        </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="SliderContainer" Storyboard.TargetProperty="Background">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderContainerBackground}" />
+                                        </ObjectAnimationUsingKeyFrames>
+                                    </Storyboard>
+                                </VisualState>
+                                 <VisualState x:Name="PointerOver">
                                     <Storyboard>
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="HorizontalTrackRect" Storyboard.TargetProperty="Fill">
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderTrackFillPressed}" />
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderTrackFillPointerOver}" />
                                         </ObjectAnimationUsingKeyFrames>
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="VerticalTrackRect" Storyboard.TargetProperty="Fill">
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderTrackFillPressed}" />
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderTrackFillPointerOver}" />
                                         </ObjectAnimationUsingKeyFrames>
-                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="HorizontalThumb" Storyboard.TargetProperty="Background">
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SliderThumbBackgroundPressed}" />
+                                        <ColorAnimationUsingKeyFrames Storyboard.TargetName="HorizontalThumb" Storyboard.TargetProperty="(Panel.Background).(SolidColorBrush.Color)">
```

Comment:
>Color [](start = 173, length = 5)

Have we tested what happens when the background isn't a solid color brush?

Reply:
I think the intention is that the outer thumb part should use a solid colour, but the inner thumb would be what the designer would change to use their brand colour.

I think a comment should be added to the template however to both the Template Part and to the Lightweight Style to hint towards the colour having no opacity

Reply:
Yes - the `Foreground` and `Background` setter properties are linked to the `SliderTrackFill`'s. The ones using ColorAnimations are not settable.

Reply:
@stmoy is this an issue that we should be concerned about?

Reply:
The potential issue is that if someone has set that panel's background brush to a non-SolidColorBrush from code behind, then taking this update would crash their app.

Reply:
How would they set the Panel's background brush from codebehind? Technically speaking you're correct that this would cause a crash, but since these properties aren't easily changed, I don't expect that folks will realistically hit this. We're taking the same targeted approach in this animation that we do with ScrollBar.